### PR TITLE
Self update

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-installation-control.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Oct 10 10:29:07 UTC 2017 - jreidinger@suse.com
+
+- add self_update_id to allow define id for self update scc query
```
Comment (mvidner) [inline package/yast2-installation-control.changes]: to allow defining an ID

File: control/control.rnc [hunk 2]
```diff
@@ -187,6 +188,9 @@ require_registration =			element require_registration { BOOLEAN }
 show_drivers_info =			element show_drivers_info { BOOLEAN }
 # Self-update (FATE#319716).
 self_update_url =			element self_update_url { text }
+# Self-update id (bsc#1057538).
+# It is id used to query scc for self update repo.
```
Comment (mvidner) [inline control/control.rnc]: It is an ID used to query SCC for the self update repo

File: package/yast2-installation-control.changes [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+  (bsc#1057538)
```
Comment (mvidner) [inline package/yast2-installation-control.changes]: Is this the correct ID? I don't see how https://bugzilla.suse.com/show_bug.cgi?id=1057538 is related.

Comment (jreidinger) [reply package/yast2-installation-control.changes]: good catch, I copy it from wrong tab, correct one is https://bugzilla.suse.com/show_bug.cgi?id=1055556

File: control/control.rng [hunk 4]
```diff
@@ -367,6 +368,15 @@ between the first and the second stage of installation</a:documentation>
       <text/>
     </element>
   </define>
+  <!--
+    Self-update id (bsc#1057538).
```
Comment (mvidner) [inline control/control.rng]: also here

File: package/yast2-installation-control.changes [hunk 5]
```diff
@@ -1,8 +1,8 @@
 -------------------------------------------------------------------
 Tue Oct 10 10:29:07 UTC 2017 - jreidinger@suse.com
 
-- add self_update_id to allow define id for self update scc query
-  (bsc#1057538)
+- add self_update_id to allow defining an id for self update scc
```
Comment (lslezak) [inline package/yast2-installation-control.changes]: NP: I'd quote "self_update_id" and use SCC (upper case)

File: package/yast2-installation-control.changes [hunk 6]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Oct 10 10:29:07 UTC 2017 - jreidinger@suse.com
+
+- add "self_update_id" to allow defining an id for self update scc
```
Comment (lslezak) [inline package/yast2-installation-control.changes]: Um, `ID` and `SCC`....

File: control/control.rnc [hunk 7]
```diff
@@ -186,10 +186,14 @@ default_ntp_setup =			element default_ntp_setup { BOOLEAN }
 enable_kdump =				element enable_kdump { BOOLEAN }
 require_registration =			element require_registration { BOOLEAN }
 show_drivers_info =			element show_drivers_info { BOOLEAN }
-# Self-update (FATE#319716).
+# Self-update URL (FATE#319716).
+## How it works?
+## here are two related keys for self update - URL and ID.
+## The ID is used to ask SCC for update URL.
```
Comment (mvidner) [inline control/control.rnc]: More picky questions: do we know the possible values/format of the ID? Or the authoritative place where it is specified?

Comment (jreidinger) [reply control/control.rnc]: @mvidner it is defined by SCC which should know which one accept. I plan to find value experimentally ( with SLES as starting point ).

**Pull request merged**